tests/src/Rector/Deprecation/GetMockRector/GetMockRectorTest.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types=1);
+
+namespace DrupalRector\Tests\Rector\Deprecation\GetMockRector;
+
+use Iterator;
+use Rector\Testing\PHPUnit\AbstractRectorTestCase;
+use Symplify\SmartFileSystem\SmartFileInfo;
+
+class GetMockRectorTest extends AbstractRectorTestCase {
+
+    /**
+     * @dataProvider provideData()
+     */
+    public function test(SmartFileInfo $fileInfo): void
+    {
+        $this->doTestFileInfo($fileInfo);
```

Comment:
This was one of the Rector rules crashing due to upstream changes. So I added a Unit test as well so we can more easily catch them without running the full functional test.